man/corosync.conf.5

```diff
@@ -572,6 +572,15 @@ Allow UDPU and KNET to drop packets from IP addresses that are not known
 (nodes which don't exist in the nodelist) to corosync.
 Value is yes or no.
 
+This feature is mainly to protect joining a nodes with outdated configuration
+after the cluster split. Another use case is to allow atomic merge of two
+independent clusters.
+
+It's not recommended to change the default value because existing cluster may
+fail when corosync is started on the unlisted node with outdated configuration.
+
+Only known drawback is a slightly reduced performance.
+
```

Comment:
How about

"Changing the default value is not recommended, the overhead is tiny and an existing cluster may fail if corosync is started on an unlisted node with an old configuration."